docs/conf.py

```diff
@@ -47,18 +45,18 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'FogLAMP'
-copyright = '2017 OSIsoft, LLC'
-author = 'OSIsoft, LLC'
+project = u'FogLAMP'
+copyright = u'2017, Dianomic Systems'
```

Comment:
OSIsoft or Dianomic?

Reply:
The documentation is owned by Dianomic
The code is owned by OSIsoft